.github/workflows/build-and-test.yml

```
@@ -90,6 +90,10 @@ jobs:
         run: |
           make genpdata
           git diff --exit-code || (echo 'Generated code is out of date, please run "make genpdata" and commit the changes in this PR.' && exit 1)
+      - name: Gen dependabot
+        run: |
+          make gendependabot
```

Alternately, can the shell be forced to Bash here?

```suggestion
          /bin/bash -c "make gendependabot"
```

---

I tested locally but the command was executed with `/bin/sh` even with `/bin/bash -c`. 
I did some investigation and I think the reason is that choosing the shell should be defined in a [makefile level](https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html).

> The program used as the shell is taken from the variable SHELL. If this variable is not set in your makefile, the program /bin/sh is used as the shell.